.github/workflows/build_changed_packages.yml

```
@@ -0,0 +1,94 @@
+name: Build changed packages
+
+on:
+  pull_request:
```

Not building on `push`?
```suggestion
  push:
  pull_request:
```

---

Thanks! That's the part I missed.